.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_DOMAIN=indexguru.us.auth0.com
```

Comment:
Add `.env` to `.gitignore` and delete the `.env` file from here, it contains sensitive details.